cmd/thanos/rule.go

```diff
@@ -614,22 +614,31 @@ func (s *alertmanagerSet) get() []*url.URL {
 
 const defaultAlertmanagerPort = 9093
 
+func parseAlertmanagerAddress(addr string) (qType dns.QType, parsedUrl *url.URL, err error) {
+	qType = ""
+	parsedUrl, err = url.Parse(addr)
+	if err != nil {
+		return qType, nil, err
+	}
+	// The Scheme might contain DNS resolver type separated by + so we split it a part
```

Comment:
the missing trailing period in the comment (: 

(We should write some linter finally for this ;p)

Reply:
:thinking: I wrote a hook for myself to Goland to check this but it's flaky on multiline comments.
But mainly I'm ignoring it apparently :smile: